package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "canonical-path": "0.0.2",
     "catharsis": "^0.8.1",
-    "change-case": "^2.1.0",
+    "change-case": "3.0.0",
```

Comment:
I don't think that this is really a breaking change. It seems that sentence-case was actually broken before - i.e. it was not capitalizing the first letter of the sentence...